server/uploader.js

```diff
@@ -34,25 +34,34 @@ uploader.DOUBLE_SIZE_ = 8;
  * @param {{
  *   type: string,
  *   name: string,
- *   fileName: string,
- *   description: string
+ *   description: string,
  * }} desc File description.
- * @param {Object} file File object received from multer.
+ * @param {!Object} file File object received from multer.
  * @param {string} prefix The destination folder to upload the file to.
  * @param {string} bigWigToWigAddr Directory of script of UCSC bigWigToWig.
  * @return {Object} Success or not as a JS Object.
  */
 uploader.uploadFile = function(desc, file, prefix, bigWigToWigAddr) {
+  var fileName = file.originalname;
+  if (fs.existsSync(prefix + fileName) ||
+    fs.existsSync(prefix + fileName + '_chr')) {
+    return {
+      error: {
+        type: 'file already exists',
```

Comment:
Usually type is a shorter keyword rather than a descriptive sentence, like 'fileExists'.